math/splot/src/TSPlot.cxx

```diff
@@ -639,12 +632,13 @@ void TSPlot::FillYpdfHists(Int_t nbins)
          return;
    }
 
-   char name[34];
    for (ispecies=0; ispecies<fNSpecies; ispecies++){
       for (i=0; i<fNy; i++){
-         snprintf(name,sizeof(name), "pdf_species%d_y%d", ispecies, i);
+         std::stringstream nameStream;
+         nameStream << "pdf_species" << ispecies << "_y" << i;
```

Comment:
Out of curiosity.  Why a `std::stringsream` instead of `operator+ for std::string` here or vice-et-versa?

Reply:
I use `operator+` when it's not too many elements that need to be concatenated, and otherwise `stringstream`.